Sources/StreamChat/ChatClient.swift

```diff
@@ -458,7 +472,7 @@ public class ChatClient {
             completion()
         }
         authenticationRepository.clearTokenProvider()
-        authenticationRepository.cancelTimers()
+        authenticationRepository.reset()
```

Comment:
It does not really matter, because this is triggered by the `reconnectionTimeoutHandler` which has `repeats: false`, so it will be already stopped by the time it reaches here